scripts/codestyle/travis_uncrustify.sh

```diff
@@ -1,5 +1,18 @@
-if ! $TRAVIS_PULL_REQUEST; then
-	bash ./run_uncrustify.sh -v $(git rev-parse HEAD~1) $(git rev-parse HEAD)
-else
-	bash ./run_uncrustify.sh -v $(git rev-parse HEAD~1) $TRAVIS_PULL_REQUEST_SHA
+#!/bin/sh
+
+git fetch origin master:master
+git fetch --depth=1000 # unlikely the branch will have more than 1000 commits
+
+revs="$(git rev-parse --short master)...$(git rev-parse --short HEAD)"
+
+echo Checking $revs
```

Comment:
`... for code style conformance` (otherwise it's hard to get it from the surrounding output)